k8s/prod/common/divorce/div-cos.yaml

```diff
@@ -36,11 +36,11 @@ spec:
         SOLICITOR_DN_REJECT_AND_AMEND: true
         PAY_BY_ACCOUNT: true
         DOCUMENT_GENERATOR_SERVICE_API_BASEURL: "http://div-dgs-java.divorce"
-        SPRING_MAIL_PROPERTIES_MAIL_SMTP_STARTTLS_ENABLE: false
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_ENABLED: true
         DATAEXTRACTION_STATUS_DA_EMAILTO: StokeCTSC4.Auto@justice.gov.uk
         DATAEXTRACTION_STATUS_DN_EMAILTO: StokeCTSC3.Auto@justice.gov.uk
         DATAEXTRACTION_STATUS_AOS_EMAILTO: Auto.StokeCTSC2@justice.gov.uk
+        DATAEXTRACTION_EMAILFROM: noreply@mail-divorce.platform.hmcts.net
```

Comment:
hmm this needs to be merged at about the same time as the other change goes live, you probably need to co-ordinate with divorce as I think they may need to update mail box rules, others teams did that